scalyr_agent/monitor_utils/annotation_config.py

```diff
@@ -174,6 +173,13 @@ def sort_numeric( pair ):
         if _is_int( key ):
             return int( key )
         return key
+
+    def key_name( key, convert_hyphens ):
```

Comment:
How about `docker_key_name` and document what this method does and why.

Reply:
`docker_key_name` is probably not appropriate as this method is also used by the k8s annotations.  Maybe `normalize_key_name` as it is used to ensure keys use the correct form - on k8s they will already contain underscores and on docker it will convert hyphens to underscores.  The function doesn't know beforehand whether it is called by docker or k8s, and relies on the `convert_hyphens` param to decide whether to convert hyphens or not.